src/workloads/query/AggregationsOutput.yml

```diff
@@ -0,0 +1,520 @@
+      Threads: 16
+      SrcCollection: *Unsharded16K
+      StartsAt: 10
+      NumCombinations: *NumCombinations
+      ShardedCollOutput: *ShardedCollOutput9
+      UnshardedCollOutput: *UnshardedCollOutput9
+
+# Sharded colls
+- ActorFromTemplate:
+    TemplateName: AggregationActor
+    TemplateParameters:
+      Threads: 1
+      SrcCollection: *Sharded1K
+      StartsAt: 11
+      NumCombinations: *NumCombinations
+      ShardedCollOutput: *ShardedCollOutput10
+      UnshardedCollOutput: *UnshardedCollOutput10
+- ActorFromTemplate:
+    TemplateName: AggregationActor
+    TemplateParameters:
+      Threads: 1
+      SrcCollection: *Sharded4K
+      StartsAt: 12
+      NumCombinations: *NumCombinations
+      ShardedCollOutput: *ShardedCollOutput11
+      UnshardedCollOutput: *UnshardedCollOutput11
+- ActorFromTemplate:
+    TemplateName: AggregationActor
+    TemplateParameters:
+      Threads: 1
+      SrcCollection: *Sharded16K
+      StartsAt: 13
+      NumCombinations: *NumCombinations
+      ShardedCollOutput: *ShardedCollOutput12
+      UnshardedCollOutput: *UnshardedCollOutput12
+- ActorFromTemplate:
+    TemplateName: AggregationActor
+    TemplateParameters:
+      Threads: 4
+      SrcCollection: *Sharded1K
+      StartsAt: 14
+      NumCombinations: *NumCombinations
+      ShardedCollOutput: *ShardedCollOutput13
+      UnshardedCollOutput: *UnshardedCollOutput13
+- ActorFromTemplate:
+    TemplateName: AggregationActor
+    TemplateParameters:
+      Threads: 4
+      SrcCollection: *Sharded4K
+      StartsAt: 15
+      NumCombinations: *NumCombinations
+      ShardedCollOutput: *ShardedCollOutput14
+      UnshardedCollOutput: *UnshardedCollOutput14
+- ActorFromTemplate:
+    TemplateName: AggregationActor
+    TemplateParameters:
+      Threads: 4
+      SrcCollection: *Sharded16K
+      StartsAt: 16
+      NumCombinations: *NumCombinations
+      ShardedCollOutput: *ShardedCollOutput15
+      UnshardedCollOutput: *UnshardedCollOutput15
+- ActorFromTemplate:
+    TemplateName: AggregationActor
+    TemplateParameters:
+      Threads: 16
+      SrcCollection: *Sharded1K
+      StartsAt: 17
+      NumCombinations: *NumCombinations
+      ShardedCollOutput: *ShardedCollOutput16
+      UnshardedCollOutput: *UnshardedCollOutput16
+- ActorFromTemplate:
+    TemplateName: AggregationActor
+    TemplateParameters:
+      Threads: 16
+      SrcCollection: *Sharded4K
+      StartsAt: 18
+      NumCombinations: *NumCombinations
+      ShardedCollOutput: *ShardedCollOutput17
+      UnshardedCollOutput: *UnshardedCollOutput17
+- ActorFromTemplate:
+    TemplateName: AggregationActor
+    TemplateParameters:
+      Threads: 16
+      SrcCollection: *Sharded16K
+      StartsAt: 19
+      NumCombinations: *NumCombinations
+      ShardedCollOutput: *ShardedCollOutput18
+      UnshardedCollOutput: *UnshardedCollOutput18
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq:
+      - shard
+      - shard-lite
+      - shard-lite-all-feature-flags
```

Comment:
Why only sharded deployments?

Reply:
The test performs the same code execution on unsharded/sharded combinations so I didn't consider normal Replica Sets for this.